integration-box/kintone/python_args.py

```diff
@@ -1,5 +1,7 @@
 import sys
 
+os.system(f"{sys.executable} -m pip install -U pytd")
```

Comment:
I guess it's a temporal workaround to drop Spark dependency before the latest Docker image becomes available, right? If so, how about passing the specific pytd version as:

```py
os.system(f"{sys.executable} -m pip install -U pytd==0.6.0")
```

Reply:
Ok, set version number explicitly 6752be2